test/infrastructure/docker/internal/docker/kind_manager.go

```diff
@@ -113,6 +114,7 @@ func (m *Manager) CreateExternalLoadBalancerNode(ctx context.Context, name, imag
 		ClusterName:  clusterName,
 		Role:         constants.ExternalLoadBalancerNodeRoleValue,
 		PortMappings: portMappings,
+		EntryPoint:   []string{"haproxy", "-sf", "7", "-W", "-db", "-f", "/usr/local/etc/haproxy/haproxy.cfg"},
```

Comment:
```suggestion
		EntryPoint:   []string{"haproxy", "-W", "-db", "-f", "/usr/local/etc/haproxy/haproxy.cfg"},
```

WDYT about trying this? AFAICT it's working fine on my machine in upgrade tests. When the config is updated haproxy logs are showing workers being spun up and down correctly to take new machines into account.

If we do this we should change the []string{} to a var at the top of the file, but apart from that I'd be happy to merge this change to assess the impact.